### PR TITLE
Add more logs for FT and journaling

### PR DESCRIPTION
**Pull request by ggezer**

### CODE REVIEW

File: core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java [hunk 1]
```diff
@@ -482,10 +483,12 @@ public synchronized void gainPrimacy() {
     mAsyncJournalWriter
         .set(new AsyncJournalWriter(mRaftJournalWriter, () -> getJournalSinks(null)));
     mTransferLeaderAllowed.set(true);
+    LOG.info("Gained primacy.");
   }
 
   @Override
   public synchronized void losePrimacy() {
+    LOG.info("Loosing primacy.");
```
Comment (yuzhu) [inline core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java:491]: Loosing? or Losing?

**Review by yuzhu — COMMENTED** [review]
nit @ggezer

File: core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java (hunk 1 above)
Comment (jenoudet) [reply core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java:491]: Losing, loose it the opposite of tight.

**Review by jenoudet — APPROVED** [review]

**Comment by Xenorith** [comment]
alluxio-bot, merge this please

**Comment by Xenorith** [comment]
alluxio-bot, cherry-pick this to branch-2.7 please

**Comment by Xenorith** [comment]
alluxio-bot, cherry-pick this to branch-2.6 please

**Pull request merged**